.github/workflows/sync-launch-files.yaml

```
@@ -28,6 +28,9 @@ jobs:
         with:
           token: ${{ steps.generate-token.outputs.token }}
           config: .github/sync-launch-files.yaml
+          pr-branch: sync-launch-files
+          pr-title: "chore: sync launch files"
+          pr-commit-message: "chore: sync launch files"
           pr-labels: |
             bot
             sync-files
```

If you want to use a different label:

```suggestion
            sync-launch-files
```

---

Thank you! Done.